tests/contrib/test_flask_util.py

```diff
@@ -384,6 +404,35 @@ def index():
             self.assertEqual(response.status_code, httplib.OK)
             self.assertTrue('Hello' in response.data.decode('utf-8'))
 
+        # Expired credentials with refresh token, should allow.
+        credentials.token_expiry = datetime.datetime(1990, 5, 28)
+        with mock.patch('oauth2client.client._UTCNOW') as utcnow:
+            utcnow.return_value = datetime.datetime(1990, 5, 29)
+
+            with self.app.test_client() as client:
+                with client.session_transaction() as session:
+                    session['google_oauth2_credentials'] =\
```

Comment:
Use parentheses rather than backslash for continuation in cases like this.

["Do not use backslash line continuation."](https://google.github.io/styleguide/pyguide.html?showone=Line_length#Line_length)